packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -554,14 +556,71 @@ private ConnectionStatusCallback requestConnectionStatusCallback(final Callback
         };
     }
 
+    @ReactMethod
+    public void showAndroidSoftKeyboard() {
+        Activity currentActivity = mReactContext.getCurrentActivity();
+        if (isAnyViewFocused()) {
+            // Cancel any previously scheduled Runnable
+            if (mKeyboardRunnable != null) {
+                currentActivity.getWindow().getDecorView().removeCallbacks(mKeyboardRunnable);
+            }
+
+            View currentFocusedView = getCurrentFocusedView();
+            if (currentFocusedView != null) {
```

Comment:
I think this condition isn't necessary because we already checked that the currently focused view is defined in line 562 via `isAnyViewFocused()`.

Reply:
Oops you're right, updated in https://github.com/WordPress/gutenberg/pull/57069/commits/1cb8d7fa13985b9c6df2e62052310e831dbc88be thanks!